hello-world/src/test/java/org/acme/schooltimetabling/solver/TimetableConstraintProviderTest.java

```diff
@@ -1,14 +1,16 @@
 package org.acme.schooltimetabling.solver;
 
+import java.time.DayOfWeek;
+import java.time.LocalTime;
+
 import ai.timefold.solver.test.api.score.stream.ConstraintVerifier;
+
 import org.acme.schooltimetabling.domain.Lesson;
 import org.acme.schooltimetabling.domain.Room;
 import org.acme.schooltimetabling.domain.Timeslot;
 import org.acme.schooltimetabling.domain.Timetable;
 import org.junit.jupiter.api.Test;
 
-import java.time.DayOfWeek;
-import java.time.LocalTime;
```

Comment:
Please reformat the imports using the order we use in the Solver.
(Alternatively: remove these unrelated changes, which will equal to the same result.)

Reply:
I believe these changes match the configuration we use in the Solver. The `java.*` imports come first.

Reply:
Ah, in that case, I must remember it wrong. OK then.